ghcide/src/Development/IDE/GHC/Util.hs

```diff
@@ -280,3 +281,10 @@ ioe_dupHandlesNotCompatible :: Handle -> IO a
 ioe_dupHandlesNotCompatible h =
    ioException (IOError (Just h) IllegalOperation "hDuplicateTo"
                 "handles are incompatible" Nothing Nothing)
+
+isOverGhc9 :: Bool
```

Comment:
We have something similar in hls-test-utils but more general, maybe it worths move the test code to check ghc version to Development.IDE.GHC.Compat and make it available for the entire codebase:

https://github.com/haskell/haskell-language-server/blob/a0def3c4236b12105970b59f791bf86df666fbcc/hls-test-utils/src/Test/Hls/Util.hs#L110-L126